PreMiD/dist/metadata.json

```diff
@@ -1,20 +1,15 @@
 {
   "author": {
-    "name": "ItsMeShisuii",
-    "id": "482142808556306446"
+    "name": "EGGSY",
+    "id": "162969778699501569"
```

Comment:
You shouldn't change author without authors permission, if you have permission to do so please let us know. 
And if you have permission to do so might be nice to add shisuii as contributor.

We need proof of the changes working,

Reply:
Sheesh, do you even check staff channels :^)

Reply:
Yes but you said after like 1min of you asking him that you probs wont reply and change anyway, you should give him a chance to reply xD

Reply:
He has time until Google reviews the extension... You won't merge this before that and the guy didn't say anything when I rewrote his code and it's been like 2 months, idk. He's probably ded, no recent messages on Discord server.

Reply:
IMO you should atleast add him back as contributor tho.

Reply:
Well none of the contributors has a word by them in the code xd I should remove all instead.